aboutme.html

```diff
@@ -36,6 +37,11 @@ <h2>Sam Mason Music</h2>
                     <img src="./images/IMG_Sammy_BW3.jpg" alt="">
                 </li>
             </ul>
+            <div class="mason-carousel-controls">
```

Comment:
This looks like it got added from a different branch perhaps from `photo-carousel` Can you please back out this change?